system/Helpers/filesystem_helper.php

```diff
@@ -75,6 +75,61 @@ function directory_map(string $sourceDir, int $directoryDepth = 0, bool $hidden
 
 // ------------------------------------------------------------------------
 
+if (! function_exists('directory_mirror'))
+{
+	/**
+	 * Recursively copies the files and directories of the origin directory
+	 * into the target directory, i.e. "mirror" its contents.
+	 *
+	 * @param string $originDir
+	 * @param string $targetDir
+	 * @param bool $overwrite   Whether individual files overwrite on collision
+	 *
+	 * @return void
+	 *
+	 * @throws InvalidArgumentException
+	 */
+	function directory_mirror(string $originDir, string $targetDir, bool $overwrite = true): void
+	{
+		$originDir = rtrim($originDir, '\\/');
+		$targetDir = rtrim($targetDir, '\\/');
+
+		if (! is_dir($originDir))
+		{
+			throw new InvalidArgumentException(sprintf('The origin directory "%s" was not found.', $originDir));
+		}
+
+		if (! is_dir($targetDir))
+		{
+			@mkdir($targetDir, 0755, true);
+		}
+
+
```

Comment:
Remove extra line

Reply:
I'm surprised Code Sniffer didn't catch this.